deploy/monitor/grafana_dashboard.yaml

```diff
@@ -0,0 +1,511 @@
+                "expr": "kube_pod_container_resource_limits_memory_bytes{container='unifiedpush-operator'}",
+                "format": "time_series",
+                "intervalFactor": 2,
+                "legendFormat": "Max Memory Allocation",
+                "refId": "C",
+                "step": 2
+              }
+            ],
+            "thresholds": [],
+            "timeFrom": null,
+            "timeRegions": [],
+            "timeShift": null,
+            "title": "Memory Usage",
+            "tooltip": {
+              "shared": true,
+              "sort": 0,
+              "value_type": "individual"
+            },
+            "type": "graph",
+            "xaxis": {
+              "buckets": null,
+              "mode": "time",
+              "name": null,
+              "show": true,
+              "values": []
+            },
+            "yaxes": [
+              {
+                "format": "bytes",
+                "label": null,
+                "logBase": 2,
+                "max": null,
+                "min": 0,
+                "show": true
+              },
+              {
+                "format": "short",
+                "label": null,
+                "logBase": 1,
+                "max": null,
+                "min": null,
+                "show": true
+              }
+            ],
+            "yaxis": {
+              "align": false,
+              "alignLevel": null
+            }
+          },
+          {
+            "aliasColors": {},
+            "bars": false,
+            "dashLength": 10,
+            "dashes": false,
+            "datasource": "Prometheus",
+            "fill": 1,
+            "gridPos": {
+              "h": 8,
+              "w": 24,
+              "x": 0,
+              "y": 18
+            },
+            "id": 2,
+            "legend": {
+              "avg": false,
+              "current": false,
+              "max": false,
+              "min": false,
+              "show": true,
+              "total": false,
+              "values": false
+            },
+            "lines": true,
+            "linewidth": 1,
+            "links": [],
+            "nullPointMode": "null",
+            "percentage": false,
+            "pointradius": 5,
+            "points": false,
+            "renderer": "flot",
+            "seriesOverrides": [],
+            "spaceLength": 10,
+            "stack": false,
+            "steppedLine": false,
+            "targets": [
+              {
+                "expr": "sum(rate(process_cpu_seconds_total{job='unifiedpush-operator'}[1m]))*1000",
+                "format": "time_series",
+                "interval": "",
+                "intervalFactor": 2,
+                "legendFormat": "UPS Operator- CPU Usage in Millicores",
+                "refId": "A",
+                "step": 2
+              },
+              {
+                "expr": "(kube_pod_container_resource_limits_cpu_cores{container='unifiedpush-operator'})*1000",
```

Comment:
@austincunningham your suggestion are impl already here. 
However, we will be working on in the Monitor in the next PRs any way. So, no worry we will be able to add it. The idea here is just adding the base for we are able to move forward. Please, are you ok with?  Could you please approve the pr now?

Reply:
Deployed your latest changes, not seeing the max allocation for the millicors and memory in the graph

ran the queries in prometheus 
kube_pod_container_resource_limits_memory_bytes{container='unifiedpush-operator'}
(kube_pod_container_resource_limits_cpu_cores{container='unifiedpush-operator'})*1000
and it returns nothing . looks like there is no container='unifiedpush-operator' ,

Reply:
Hi @austincunningham really tks for ALL your amazing help with.
As we chat w don't know why but this metrics has been not exported by default in this oper so we decided just remove and go forward.